frontend/src/app/App.tsx

```diff
@@ -39,7 +39,7 @@ const App: FC = () => {
         <Modal />
         <PageLoader />
         <Routes>
-          <Route element={<ProtectedRoutes roles={[Roles.COS_OFFICER]} />}>
+          <Route element={<ProtectedRoutes roles={[Roles.CEEB]} />}>
```

Comment:
@dmitri-korin-bcps this needs to include both COS_OFFICER and CEEB tho. Otherwise, COS users will be redirect to /not-authorized again